src/components/Charts.tsx

```diff
@@ -2,6 +2,8 @@ import React from "react";
 import { Chart, AxisOptions, AxisOptionsBase } from "react-charts";
 import { Result } from "roamjs-components/types/query-builder";
 
+type ChartData = [Result[string], Result[string]];
```

Comment:
this is an example of what not to do in a PR.

The upgrading of `tsconfig.json` (https://github.com/dvargas92495/roamjs-query-builder/pull/55/files#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0) that happened as a byproduct of going from `roamjs-scripts` to `@samepage/scripts` meant a bunch of typescript rules that weren't previously enforced are now enforced, which led to a bunch of cleanup work throughout the PR.

I'll highlight any relevant changes that weren't a part of this.